app/views/maestros/show.html.erb

```diff
@@ -0,0 +1,9 @@
+<h3><%= @maestro.artist %></h3>
+
+  <p><%= @maestro.album %></p>
+
+  <p><%= @maestro.genre %></p>
+
+  <p><% if @maestro.favorite %> (This is MY JAM!!!) <% end %></p>
```

Comment:
if it is not a favorite, then there will be an empty paragraph. Instead, consider wrapping the entire paragraph with the `if` statement